sqlvm-alwayson-cluster/nested/provisioningVM1.json

```diff
@@ -0,0 +1,412 @@
+{  
```

Comment:
There's a better way to dynamically select data disks rather than maintaining 4 separate template files. See https://github.com/Azure/azure-quickstart-templates/tree/master/201-vm-dynamic-data-disks-selection